doc/articles/uno-packages.md

```diff
@@ -0,0 +1,39 @@
+| [Uno.CodeGen](https://www.nuget.org/packages/Uno.CodeGen) | [Uno.CodeGen](https://www.nuget.org/packages/Uno.CodeGen) |  | Provides tooling for code generation. |
+| [Uno.Fonts.Fluent](https://www.nuget.org/packages/Uno.Fonts.Fluent) | [Uno.Fonts.Fluent](https://www.nuget.org/packages/Uno.Fonts.Fluent) |  | Uno Platform Fluent Font (Windows 11 iconography). |
+
+## Uno.Themes
+
+| WinUI                                                                     | UWP                                                                 | Documentation                                                                             | Description                                                                                                           |
+|---------------------------------------------------------------------------|---------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| [Uno.Material.WinUI](https://www.nuget.org/packages/Uno.Material.WinUI)   | [Uno.Material](https://www.nuget.org/packages/Uno.Material)         | https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html  | Library designed to help with the use of Material Design 3 by Google in Uno Platform.                                 |
+| [Uno.Cupertino.WinUI](https://www.nuget.org/packages/Uno.Cupertino.WinUI) | [Uno.Cupertino](https://www.nuget.org/packages/Uno.Cupertino)       | https://platform.uno/docs/articles/external/uno.themes/doc/cupertino-getting-started.html | Uno package for Cupertino by Google. | Library designed to help with the use of Cupertino - Human Interface by Apple. |
+
+## Uno.Extensions
+
+| WinUI                                                                     | UWP                                                                 | Documentation                                                                             | Description                                                                                                           |
+|---------------------------------------------------------------------------|---------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| [Uno.Extensions](https://www.nuget.org/packages/Uno.Extensions)   | [Uno.Extensions](https://www.nuget.org/packages/Uno.Extensions)         | https://platform.uno/docs/articles/external/uno.extensions/doc/ExtensionsOverview.html | Extensions to accelerate your application development with Uno Platform, UWP and WinUI. |
```

Comment:
@nickrandolph Is `Uno.Extensions` NuGet package actively used, or only `Uno.Extensions.*` packages?